lib/internal/assert/assertion_error.js

```diff
@@ -161,7 +154,9 @@ function createErrDiff(actual, expected, operator) {
     return `${kReadableOperator.notIdentical}\n\n${actualLines.join('\n')}\n`;
   }
 
-  if (i > 3) {
+  // There were at least five identical lines at the end. Mark a couple of
+  // skipped.
+  if (i > 4) {
```

Comment:
Nit, I would prefer `i >= 5` instead of `i > 5` or would make the comment "more than four" rather than "at least five"